server/db/models/song.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+var _ = require('lodash');
+var Sequelize = require('sequelize');
+
+module.exports = function (db) {
+
+     db.define('songs', {
+        title: {
+            type: Sequelize.STRING,
+            allowNull: false
+        },
+        description: {
+            type: Sequelize.TEXT,
+            allowNull: false
```

Comment:
It wouldnt have to be long descriptions, i can always just come up with something similar off the top of my head. We can leave it in there